src/pages/tutorial/react/step-1.mdx

```diff
@@ -23,7 +23,7 @@ import Preview from 'components/Preview';
 
 ## Preview
 
-A [preview](https://react-step-2--carbon-tutorial.netlify.com) of what you will build:
+A [preview](https://react-step-2--carbon-tutorial.netlify.com) of what you will build if you are lucky:
```

Comment:
```suggestion
A [preview](https://react-step-2--carbon-tutorial.netlify.com) of what you will build:
```

This change seems unrelated to the rest of the PR, removing for now.

Reply:
hahahahahha that came through from when we did our test one - thank goodness you spotted it!